packages/components/src/components/tooltip/tooltip.hbs

```diff
@@ -5,17 +5,21 @@
   LICENSE file in the root directory of this source tree.
 -->
 
-<div id="tooltip-label" class="{{@root.prefix}}--tooltip__label" aria-describedby="unique-tooltip">
+{{!-- Using the trigger label as the tooltip label (can be repetitive for screen reader users) --}}
+<div id="tooltip-label" class="{{@root.prefix}}--tooltip__label">
   Tooltip label
+  {{!-- @todo add aria-expanded once JavaScript can toggle it --}}
+  {{!-- @todo use a button element once JavaScript can support it as the trigger (remove tabindex="0" and role="button") --}}
   <div tabindex="0" aria-labelledby="tooltip-label" data-tooltip-trigger data-tooltip-target="#unique-tooltip"
-    role="tooltip" class="{{@root.prefix}}--tooltip__trigger">
+    role="button" class="{{@root.prefix}}--tooltip__trigger" aria-controls="unique-tooltip" aria-haspopup="true">
     {{ carbon-icon 'Information16' }}
   </div>
 </div>
+{{!-- @todo add aria-hidden once JavaScript can toggle it --}}
 <div id="unique-tooltip" data-floating-menu-direction="bottom" class="{{@root.prefix}}--tooltip"
-  data-avoid-focus-on-open>
+  data-avoid-focus-on-open role="dialog" aria-describedby="unique-tooltip-body" aria-labelledby="tooltip-label">
   <span class="{{@root.prefix}}--tooltip__caret"></span>
-  <p>This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is
+  <p id="unique-tooltip-body" >This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is
```

Comment:
Could you generate unique ID in `.config.js` and refer to that here? Thanks!

Reply:
Hi @asudoh @elizabethsjudd  could these be UUID? [node-uuid](https://github.com/kelektiv/node-uuid) could be used for this, generating unique keys for each demo (so they don't override each other.) Would require a new dependency, but node-uuid is the most popular/supported tool for this.

Reply:
@scottnath Any solutions generating unique ID is fine. e.g. This is our dev-time only I wouldn't foresee a collision with `Math.random()`, etc.

Reply:
@asudoh confused by the message. Does that mean it's ok to add the package [node-uuid](https://github.com/kelektiv/node-uuid) and to use it to create ID's with uuid? This package would then be available to any component that needs to create unique ID's

Reply:
Using `node-uuid` is OK, but I wouldn't given there is an easier way. Also if you are strong for `node-uuid` please use it with `devDependencies`.